py/src/nupic/encoders/rdse.py

```diff
@@ -70,6 +73,14 @@
     parameters.size       = args.size
     parameters.sparsity   = args.sparsity
 
+    if args.category:
+        if parameters.radius == 0:
```

Comment:
this is a great change! Rather than "just" in a demo program, 
make this available to both c++,py encoders? 
Please consider adding flag `bool onlyCategory` to RDSE,Scalar+bidnings constructors.

Reply:
That's a good idea.  I will add the flag to the parameter structure.  Then the encoder can check for radius==1 and enforce integer only inputs.  I will do these things before merging.  Thanks!